cmd/opm/alpha/bundle/unpack.go

```diff
@@ -70,14 +75,30 @@ func unpackBundle(cmd *cobra.Command, args []string) error {
 	}
 
 	var (
-		registryOpts []containerdregistry.RegistryOption
-		skipTLS      bool
+		registryOpts  []containerdregistry.RegistryOption
+		useHTTP       bool
+		skipTLSVerify bool
+		skipTLS       bool
 	)
 	skipTLS, err = cmd.Flags().GetBool("skip-tls")
 	if err != nil {
 		return err
 	}
-	registryOpts = append(registryOpts, containerdregistry.SkipTLS(skipTLS))
+	skipTLSVerify, err = cmd.Flags().GetBool("skip-tls-verify")
+	if err != nil {
+		return err
+	}
+	useHTTP, err = cmd.Flags().GetBool("use-http")
+	if err != nil {
+		return err
+	}
+	if skipTLS {
+		// Set useHTTP when use deprecated skipTlS
+		// for functional parity with existing
+		useHTTP = true
+	}
```

Comment:
What happens if someone uses `--skip-tls=true --use-http=false`? Should we error out in that case?

Maybe even better, should we just make `skip-tls` mutually exclusive from both `skip-tls-verify` and `use-http`?

Reply:
Also, it seems like `--skip-tls-verify=false --use-http=true` is an invalid combination, right?

Reply:
Agree with doing the flag combination checks. My intent here was to make it simpler to rip out `skip-tls` eventually because it is being deprecated here, but I see how doing that might introduce some unexpected changes since it may not always equate to `use-http` in every case. I will make that change.

Reply:
Not sure if it's possible, but given this code is repeated in so many places, it might be worth taking a look at writing a helper function somewhere in the `cmd` tree (perhaps `cmd/internal/util` or something?) so it can be reused everywhere.